logging-utils/samples/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "google-logging-samples",
+  "version": "0.0.1",
+  "description": "Samples for adhoc debug logger",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "compile": "tsc -p .",
+    "typeless": "npx typeless-sample-bot --outputpath javascript --targets typescript --recursive",
+    "posttypeless": "npx eslint --fix javascript",
+    "preinstall": "npm link ../"
+  },
+  "files": [
+    "javascript/*.js",
+    "typescript/*.ts"
+  ],
+  "private": true,
+  "author": "Google API Authors",
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/googleapis/gax-nodejs.git",
+    "directory": "adhoc-logging/samples"
+  },
+  "engines": {
+    "node": ">=14"
+  },
+  "dependencies": {
+    "@google-cloud/adhoc-logging": "0.0.1"
```

Comment:
should these be updated to say `logging-utils` and not `adhoc-logging`?

Reply:
It should be `google-logging-utils` (no google-cloud/). Good catch.